File: java/src/test/compile-fail/PackWithoutScheme.java
package packbin;

final class PackWithoutScheme {
    static final class MarkerRow {
        public byte sid;
    }

    public static void main(String[] args) {
        MarkerRow row = new MarkerRow();
        row.sid = 23;
        BinaryPacker.pack(row);
    }
}

File: csharp/tests/compile-fail/PackWithoutScheme.cs
using Packbin;

public sealed class MarkerRow
{
    public byte Sid { get; set; }
}

public static class Program
{
    public static void Main()
    {
        var row = new MarkerRow { Sid = 23 };
        _ = BinaryPacker.Pack(row);
    }
}
